### ISSUE TITLE
swcarpentry/make-novice

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#75]
tracking issue for <https://github.com/swcarpentry/make-novice>

 - [x] fix filter for image
 - [ ] fix syntax error (intro.md:313 should be reference link)
 - [x] fix reference definition lists
 - [x] fix setup location
 - [x] move `code/` folder to `files/code/`
 - [x] fix all episode links with `code/` to be `files/code/`
 - [ ] update `commands.mk` at top of lesson so that targets point to `episodes/files/code/` and `episodes/fig/`

```markdown
episodes/01-intro.md:313 [missing file]: [well-documented](gnu-make-documentation)
episodes/02-makefiles.md:412 [uninformative link text]: [here](code/02-makefile/Makefile)
episodes/09-conclusion.md:70 [missing file]: [](fig/09-conclusion-challenge-1.png)
index.md:51 [missing file]: [setup](setup.md)
```

**Comment by zkamvar** [comment]
A note for @gcapes: The [folder structure will change](https://carpentries.github.io/workbench/transition-guide.html#folder-structure) which means that you may be a little disoriented when you try to maintain any of the auxilary files for the lesson (e.g. code, figures, data). I've done my best to fix internal links so that they work, but there may still be places that I missed.

The differences that you will see:

| styles |  workbench |
| ------ | ---------- |
| data/  | episodes/data/       |
| fig/   | episodes/fig/        |
| code/  | episodes/files/code/ |

The reasoning behind this shift is the notion that the lesson centers around the episodes. Anything that are related to episodes can be categorized as data, figures or other files (e.g. code, but could also be handouts). Having these live inside the episodes folder means that we can avoid the paradigm of working in one folder and having to think about the final website structure.

This means that you might need to adjust how you create your figures and make sure the commands.mk file is accurate.

**Comment by zkamvar** [comment]
This is closed in e131513, but note the two issues remaining.

I could not fix the first issue because the reference link did not exist and I did not want to attempt to fix the second for fear that it would hinder repairing it if I did not correctly fix it. 

Neither of these are urgent.

**Comment by gcapes** [comment]
I've fixed the first - just need write access now :)
